cmd/members/invite.go

```diff
@@ -0,0 +1,88 @@
+package members
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"ldcli/cmd/cliflags"
+	resourcescmd "ldcli/cmd/resources"
+	"ldcli/cmd/validators"
+	"ldcli/internal/errors"
+	"ldcli/internal/members"
+	"ldcli/internal/output"
+	"ldcli/internal/resources"
+)
+
+func NewMembersInviteCmd(client resources.Client) *cobra.Command {
+	cmd := &cobra.Command{
+		Args:  validators.Validate(),
+		Long:  "Create new members and send them an invitation email",
+		RunE:  runE(client),
+		Short: "Invite new members",
```

Comment:
see comment in pr description: this is the same as what we have for `create`, which comes straight from the openapi spec. Not sure what else we might want to put here instead.